bundles/org.openhab.binding.vesync/src/main/java/org/openhab/binding/vesync/internal/handlers/VeSyncBridgeHandler.java

```diff
@@ -198,7 +218,7 @@ public Collection<Class<? extends ThingHandlerService>> getServices() {
 
     @Override
     public void initialize() {
-        api.setHttpClient(httpClientProvider.getHttpClient());
+        api.setHttpClient(httpClient);
```

Comment:
If the `VeSyncV2ApiHelper` has a constructor that accepts the httpclient, it would make the VeSyncV2ApiHelper .httpClient member non-null and it can be instantiated when this class 's constructor is called.